automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -76,20 +69,22 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
         val user = UserPool.chooseAuthDomainUser
         implicit val authToken: AuthToken = authTokenDefault
         withGroup("AuthDomain", List(user.email)) { authDomainName =>
-          withWorkspace(projectName, "AuthDomainSpec_share", Set(authDomainName), List(AclEntry(user.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
-            withCleanUp {
-              withSignIn(user) { listPage =>
-                val summaryPage = listPage.enterWorkspace(projectName, workspaceName)
+          withCleanBillingProject(defaultUser) { projectName =>
+            withWorkspace(projectName, "AuthDomainSpec_share", Set(authDomainName), List(AclEntry(user.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
+              withCleanUp {
+                withSignIn(defaultUser) { listPage =>
```

Comment:
If I remember correctly, the original reason for the additional user in this test was to verify specifically that a user other than the workspace owner can not widen the auth domain by cloning the workspace. While the same is true for the workspace owner, I think it's a better test if done by another user. Is there anything preventing us from doing:
```
withCleanBillingProject(defaultUser, List(user.email)) { ...
```
above and keeping the users as they were? Same applies to the 2 cases below related to multi-group auth domains.

If there is good reason for this change, the test should be simplified by removing the extra user altogether.